### PR TITLE
refresh homepage

### PR DESCRIPTION
**Pull request by deepakgithub11**
Upcoming changes for WPUI Home Page

### CODE REVIEW

**Comment by deepakgithub11** [comment]
for now flag png is used will replace later with svg once svg is available.

**Comment by deepakgithub11** [comment]
IMG replaced with SVG
changes done for logo SVG
all Figma changes applied

File: src/components/banner/style.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.wpui-lubus{
+    background-color: rgba(78, 78, 78, 0.38);
+    padding: 7px;
+    padding-left: 12px;
+    padding-right: 12px;
```
Comment (ajitbohra) [inline src/components/banner/style.scss:5]:
```suggestion
    padding: 7px 12px;
```

we can use the shorthand

File: src/components/footer/index.js [hunk 2]
```diff
@@ -17,6 +17,10 @@ import {
  * Internal dependencies.
  */
 import './style.scss'
+import { Flag, Logo } from '../../components'
+
+const date = new Date();
+let Year = date.getFullYear();
```
Comment (ajitbohra) [inline src/components/footer/index.js:23]:
```suggestion
const Year = date.getFullYear();
```

File: src/components/footer/index.js [hunk 3]
```diff
@@ -25,20 +29,26 @@ function Footer() {
     return (
         <Card className="wpui-footer" size="large" isBorderless>
             <CardBody>
-                <VStack className="sec-1" alignment="center" spacing={8}>
-                    <Heading color='white'>WPUI Components</Heading>
+                <VStack className="sec-1" alignment="center" spacing={4}>
```
Comment (ajitbohra) [inline src/components/footer/index.js:32]:
```suggestion
                <VStack className="footer-content" alignment="center" spacing={4}>
```

class can be more descriptive!

File: src/components/footer/index.js [hunk 4]
```diff
@@ -25,20 +29,26 @@ function Footer() {
...
                     </Button>
                 </VStack>
             </CardBody>
             <CardDivider style={{ backgroundColor: '#ffffff4f' }} />
-            <CardBody>
-                <HStack justify='center'>
-                    <Text color='white'>Proudly made in India by LUBUS</Text>
+            <CardBody style={{ padding: '20px' }}>
+                <HStack className='wpui-footer-bottom'>
+                    <Text className='wpui-footer-bottoml' color='white'>&copy; {Year} WPUI. All Rights Reserved.</Text>
```
Comment (ajitbohra) [inline src/components/footer/index.js:47]:
```suggestion
                    <Text className='wpui-footer-bottom-left' color='white'>&copy; {Year} WPUI. All Rights Reserved.</Text>
```

File: src/components/footer/index.js [hunk 5]
```diff
@@ -25,20 +29,26 @@ function Footer() {
...
+                    <HStack expanded={false} spacing={1}  className='wpui-footer-bottomr'>
```
Comment (ajitbohra) [inline src/components/footer/index.js:48]:
```suggestion
                    <HStack expanded={false} spacing={1}  className='wpui-footer-bottom-right'>
```

**Review by ajitbohra — CHANGES_REQUESTED** [review]
need a few code changes, functionality wise looks good.

**Pull request merged**